build.gradle

```diff
@@ -174,12 +176,58 @@ idea {
     }
 }
 
+task sourceJar(type: Jar, dependsOn: classes) {
+  from sourceSets.main.allJava
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+  from javadoc.destinationDir
+}
+
+publishing {
+  publications {
+    bintray(MavenPublication) {
+      from components.java
+      groupId project.group
+      artifactId project.name
+      version project.version
+
+      artifact sourceJar {
+        classifier 'sources'
+      }
+
+      artifact javadocJar {
+        classifier 'javadoc'
+      }
+    }
+  }
+}
+
 shadowJar {
     if (jdk != 'jdk9') {
         from project.files(toolsLocation)
     }
 }
 
+bintray {
+  user = rootProject.hasProperty('bintrayUser') ? rootProject.property('bintrayUser') : 'FIXME'
+  key = rootProject.hasProperty('bintrayApiKey') ? rootProject.property('bintrayApiKey') : 'FIXME'
+  publications = ['bintray']
+
+  pkg {
+    repo = 'maven'
+    licenses = ['BSD 2-Clause']
+    vcsUrl = 'https://github.com/AdoptOpenJDK/jitwatch'
+    name = "${project.group}:${project.name}"
+
+    version {
+      name = project.version
+      released = Date.from(java.time.Instant.now())

```

Comment:
Egregious use of JSR 310 because this is the shortest way I know to get a Date in UTC